src/main/java/com/splunk/ibm/mq/opentelemetry/Main.java

```diff
@@ -48,7 +48,7 @@ public static void main(String[] args) {
 
     int numberOfThreads = 1;
     int taskDelaySeconds = 60;
-    int initialDelaySeconds = 10;
+    int initialDelaySeconds = 0;
```

Comment:
@atoulme  I assume this is tested and works!  :-)

Reply:
yes, I tested it.